README.md

```diff
@@ -84,24 +84,23 @@ For **ARM Devices** (such as Raspberry Pi) things might be a bit different. If y
 1. From that recovery image, you will need the Widevine files located in /opt/google/chrome/libwidevinecdm*.so.
 1. These files need to be copied to the `<kodi-profile>/cdm` folder.
 
-_NOTE: Keep in mind that you might need to try multiple recovery images before you find a working one._ 
+_Note: Keep in mind you might need to try multiple recovery images before you find a working one._ 
 
 # Acknowledgement #
-The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by
-BaKMaN.
+The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by BaKMaN.
 
 # Copyrights and Licenses #
-Retrospect is licensed under a _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
+Retrospect is licensed _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
 
-The official add-on package for Retrospect may not be distributed via other repositories than the official Kodi add-on repository.
+The official add-on package for Retrospect is not distributed via other repositories than the official Kodi add-on repository.
```

Comment:
This actually means "you are not allowed to distribute it otherwise than the Kodi repo"

Reply:
No, it just means those aren't official ones, which holds true.

Reply:
Nope, "may not" means what Bas says. However, I would suggest just "Retrospect" as it's self-evident it's the package.

Reply:
The original or the new version? It is permitted by the license, and then it becomes a question of what is official and what is not.